src/main/java/com/hcl/appscan/jenkins/plugin/auth/JenkinsAuthenticationProvider.java

```diff
@@ -102,9 +102,10 @@ private String getToken() {
 		return Secret.toString(m_credentials.getToken());
 	}
 
-	public boolean isAppScan360(){
-		String url = m_credentials.getUrl();
-        	return  url != null && !url.isEmpty() && !url.contains("appscan.com");
+	@Override
```

Comment:
Is this required to be override ? Can we add this is only in Jenkins Auth.

Reply:
As we are now using the isAppScan360() method in the appscan-plugin instead of the SDK. I will not go with overriding this method.